stage1/init/init.go

```diff
@@ -650,6 +655,34 @@ func stage1() int {
 	return 0
 }
 
+func mountHostCgroups() error {
```

Comment:
It needs some comments to explain why it is necessary. Mention that systemd-nspawn requires a cgroup named `name=systemd` and that some distributions such as ... don't provide it.